pyccel/parser/semantic.py

```diff
@@ -1317,7 +1317,6 @@ def _ensure_target(self, rhs, d_lhs):
         if isinstance(rhs, IndexedElement) and rhs.rank > 0 and rhs.base.internal_variable.allocatable:
             d_lhs['allocatable'] = False
             d_lhs['is_pointer' ] = True
-
```

Comment:
Is there any reason for removing whitespace? Spacing out code can help make it more readable. I have seen several PRs recently where some people remove whitespace and others put it back

Reply:
i didn't notice that i removed it mb.

Reply:
There are quite a lot of removed/added whitespace ^^'